crypto/rsa/rsa_backend.c

```diff
@@ -172,6 +172,8 @@ int ossl_rsa_fromdata(RSA *rsa, const OSSL_PARAM params[], int include_private)
                  * since we set our key and 2 factors above we can skip
                  * the call to ossl_rsa_set0_all_params
                  */
+                if (ctx == NULL)
+                    goto err;
```

Comment:
I think the more correct fix here is to move the allocation of the ctx to the top of the function from line 102, and check for null there

Reply:
Actually this really is _cannot happen_ condition. The allocation of the ctx is implied by include_private but is_private is implied by include_private as well.

To make things less confusing for static analysers, I'd suggest moving the statement `is_private = (d != NULL);` inside the `if (is_private)` block. That will make it clear that there is nothing to check here as it was already checked on line 103.